services/jupyterhub/Dockerfile

```diff
@@ -1,13 +1,6 @@
-FROM jupyterhub/jupyterhub:0.9
+FROM jupyterhub/jupyterhub:0.8.1
```

Comment:
how about:

```
ARG JUPYTERHUB_VERSION=master
FROM jupyterhub/jupyterhub:$JUPYTERHUB_VERSION
```

this variable should be set in the `Makefile` and a target for jupyterhub in the `Makefile` should use the `--build-arg` flag to override the default.